R/load.R

```diff
@@ -876,12 +882,19 @@ renv_load_report_synchronized <- function(project = NULL, lockfile = NULL) {
   if (length(intersect(lockpkgs, libpkgs)) == 0 && length(lockpkgs) > 0L) {
 
     caution("- None of the packages recorded in the lockfile are currently installed.")
-    if (renv_rstudio_autoloading()) {
-      caution("- Use `renv::restore()` to restore the project library.")
+    autoloading <- getOption("renv.autoloader.running", default = FALSE)
+    if (autoloading) {
+      if (requireNamespace("cli", quietly = TRUE)) {
+        cli::cli_bullets(c(
```

Comment:
`renv` intentionally doesn't try to load any packages (or use any custom packages), as we don't want to cause issues for users on Windows who might want to install or update these packages.

Can we use own our `renv` shim here? Could this just be `caution()`? How hard would it be for us to implement support for hyperlinks ourselves in `renv`?

Reply:
I'm going to remove this whole attempt to provide a clickable (as in, runnable) hyperlink. There's the point you make but also the fact that it's logically impossible to hit this code (explained elsewhere).

This effort made me appreciate that you really can't use other packages to create the UI for renv itself.